### PR TITLE
[Y2K-406] format all phone numbers in search

### PR DESCRIPTION
**Pull request by jzila**
cc @keybase/y2ksquad

### CODE REVIEW

**Comment by jzila** [comment]
Got it, I'll change it.

File: shared/util/phone-numbers/index.tsx [hunk 1]
```diff
@@ -116,19 +117,29 @@ export const validateNumber = (rawNumber: string, region?: string | null) => {
...
-  const number = phoneUtil.parse(rawNumber, '')
-  return `+${number.getCountryCode()} ${phoneUtil.format(number, PNF.NATIONAL)}`
+  const phoneNumber = phoneUtil.parse(rawNumber, '')
+  return `+${phoneNumber.getCountryCode()} ${phoneUtil.format(phoneNumber, PNF.NATIONAL)}`
+}
+
+export const formatAnyPhoneNumbers = (rawText: string) => {
+  const found = rawText.match(/(\+)?(\d)+/)
+  const rawNumber = found ? found[0] : ''
+  const validatedNumber = validateNumber(rawNumber)
```
Comment (zapu) [inline shared/util/phone-numbers/index.tsx]: should it also early exit here if a phone number was not found?

File: shared/util/__test__/phonenumbers.test.tsx [hunk 2]
```diff
@@ -3,33 +3,33 @@ import {validateNumber, formatPhoneNumber, e164ToDisplay} from '../phone-numbers
...
     // are in, provided they use full E164 format with country code.
-    expect(validateNumber('+48784123123', null)).toEqual({e164: '+48784123123', valid: true})
-    expect(validateNumber('+48784123123', 'US')).toEqual({e164: '+48784123123', valid: true})
-    expect(validateNumber('+12015551113', 'PL')).toEqual({e164: '+12015551113', valid: true})
+    expect(validateNumber('+48784123123', null)).toMatchObject({e164: '+48784123123', valid: true})
+    expect(validateNumber('+48784123123', 'US')).toMatchObject({e164: '+48784123123', valid: true})
+    expect(validateNumber('+12015551113', 'PL')).toMatchObject({e164: '+12015551113', valid: true})
   })
 
   it('formats phone numbers correctly', () => {
```
Comment (zapu) [inline shared/util/__test__/phonenumbers.test.tsx]: do you also want to add test for formatAnyPhoneNumbers specifically?

**Comment by zapu** [comment]
oh it's already merged after I refreshed, well disregard the comments then

File: shared/util/phone-numbers/index.tsx (hunk 1 above)
Comment (jzila) [reply shared/util/phone-numbers/index.tsx]: It does early exit...note the `!phoneNumber` in the if statement below.

**Pull request merged**